xray/services/scan.go

```diff
@@ -165,50 +169,12 @@ type XrayGraphScanParams struct {
 	ProjectKey             string
 	Watches                []string
 	ScanType               ScanType
-	Graph                  *xrayUtils.GraphNode
+	AuditGraph             *xrayUtils.GraphNode
+	BinaryGraph            *xrayUtils.BinaryGraphNode
```

Comment:
I understand that you want to provide a clarification where one graph is used in the Audit command, and the second one refers to Binaries. So, what do you think about changing the names to: DependenciesGraph and BinariesGraph? In my opinion, the name "AuditGraph" might not be the most appropriate choice, as it might not directly convey that this graph specifically contains dependencies. What do you think?